en/docs/references/test-your-service-via-postman.md

```diff
@@ -0,0 +1,147 @@
+# Test Your Service via Postman
+
+This guide explains how to test your services by connecting them to your Postman account via an API key.
+
+## Prerequisites
+
+To use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
```

Comment:
Since we've used "sign in" to Choreo Console in almost all other. places, shall we use the same here too?
Also need to use capital C for Console.